content/soda-gettingstarted/multicloud-ha-installation-using-ansible.md

```diff
@@ -66,9 +68,13 @@ ansible-playbook site.yml -i local.hosts
 ansible-playbook site.yml -i local.hosts -vvv
 ```
 
-The namespace 'soda-multi-cloud' is used deploy the services in the Kubernetes Cluster.
-After deploying check the Pods, Deployments and Services deployed in the Kubernetes Cluster. 
-Use the command "kubectl get pods -n soda-multi-cloud" to get a list of all the pods deployed on the Kubernetes Cluster. For more kubect commands please refer to https://kubernetes.io/docs/reference/kubectl/
+Here is a block diagram showing the deployment of SODA multi-cloud services in HA mode. 
+ ![SODA Mulit-Cloud HA Deployment](resources/multicloud-ha-fig1.png)
```

Comment:
Mulit -> Multi
Did you upload the image? seems like image is not found when the site is getting deployed.

Reply:
its a bin file..

Reply:
I can see it in the local preview 
![image](https://user-images.githubusercontent.com/65751939/113406749-a623b680-93c9-11eb-8532-2498c3a6371e.png)